devdocs/process.md

```diff
@@ -82,8 +82,10 @@ This workflow is an adaptation of the [GitHub flow](https://guides.github.com/in
      doesn't always result in a good layout.
         
    * Ensure _dev green_ (i.e., all tests are passing on dev server).
+
+    >If any of the browsertests fail, use [**GodMode**](/devdocs/godmode.md) to fix them. Run the test suite again without GodMode and ensure _dev green_
```

Comment:
Perhaps the second sentence is more accurate as "Ensure that dev green is reached without the need for GodMode before submitting for review".